content/influxdb/v1.8/query_language/explore-schema.md

```diff
@@ -438,10 +438,7 @@ See the Data Exploration page for documentation on the
 [`OFFSET` clause](/influxdb/v1.8/query_language/explore-data/#the-offset-clause),
 and on [Regular expressions in queries](/influxdb/v1.8/query_language/explore-data/#regular-expressions).
 
-The following queries with wildcards characters (`*`) are also allowed:
-
-- `SHOW MEASUREMENTS ON *.*` to show all databases and all RPs
-- `SHOW MEASUREMENTS ON <db>.*` to show a specific database and all its RPs
+Run `SHOW MEASUREMENTS ON <db>.*` to show a specific database and all its RPs.
```

Comment:
I think this line has to go as well.  @lesam ?

Reply:
Yes. Previous behaviour, preserved in all versions:

```
show measurements # only valid from CLI after use database, same as:
show measurements on <database> # shows all measurements from all retention policies, not broken out by policy
```

Additional behaviour in OSS 2.1.0 and Enterprise 1.10:
```
show measurements on *.* # all databases and all retention policies, broken out by retention policy
show measurements on <db>.* # specific database and all RPs, broken out by retention policy (different than non-wildcard which is not broken by retention policy, for backward-compatibility)
show measurements on <db>.<rp> # specific database and specific RP
```

Error everywhere:
```
show measurements .*
show measurements on .*
```